### ISSUE TITLE
[MAS1.3.1]: Ensures elements with an ARIA role that require child roles contain them. 

### ISSUE THREAD

**Issue by bcoard** [microsoft/fluentui#13407]
User Impact: 
People with Visual Impairments will be unable to know the group/header name while navigated to the items under it.

Build Details: ​
OS Version: 19628.1
AT: Narrator

Repro Steps: ​
Using a grouped list run A11yInsights Fast Pass.

Actual Result: ​
Required ARIA children role not present: group label must be associated to all the list items in the list under them.

Expected Result: ​
Ensures elements with an ARIA role that require child roles contain them 

This image for a list in use today but the same failure should apply with the out of box experience.

![image](https://user-images.githubusercontent.com/7798177/94976782-e6fa4b80-04ca-11eb-8b5e-02161000cafe.png)

**Comment by khmakoto** [comment]
This is something that is occurring also on our `GroupedList.Custom.Example.tsx`, but after some investigation it turns out that those roles need to be provided `onRenderCell` which is something the user must pass. I'll fix the example, but to fix the actual error you should provide `role="row"` and `role="cell"` in your `onRenderCell`.

**Comment by bcoard** [comment]
Thanks. Could we add some documentation on this as well? Could also explain, possibly, why this can't be fixed in the control itself?

**Comment by khmakoto** [comment]
Basically, `GroupedListSections` render a `List` of role `grid` with its cells determined by what `onRenderCell` is passing. A `grid` role expects to have as children elements with either role `row` or `rowgroup`. An element with type `row` in turn expects to have children with roles `cell`, `columnheader`, `gridcell` or `rowheader`.

Since `onRenderCell` is something provided entirely by the user the responsibility of applying this roles relies on them if they choose to not use the `DetailsRow` component to render those cells.

**Comment by bcoard** [comment]
@khmakoto I've managed to get a repro in CodePen even with your changes.

First collapse all the headers in this CodePen then run Accessibility Insights.
https://codepen.io/bcoard/pen/BajNEvL?editors=0010

**Comment by khmakoto** [comment]
This issue seems to have been fixed by #14835. I have talked with @bcoard offline about it and he'll check things on his end to see if he's pulling the correct version to get these changes.
